www/front_src/src/Authentication/Openid/models.ts

```diff
@@ -13,40 +13,84 @@ export interface AuthorizationRelationToAPI {
   claim_value: string;
 }
 
+export interface Relations {
+  accessGroup: NamedEntity;
+  claimValue: string;
+}
+
+export interface RelationsToAPI {
+  access_group_id: number;
+  claim_value: string;
+}
+
+export interface EndpointToAPI {
+  custom_endpoint: string | null;
+  type: string;
+}
+
+export interface Endpoint {
+  customEndpoint: string | null;
+  type: string;
+}
+
+export interface RolesMapping {
+  apply_only_first_role: boolean | null;
+  attribute_path: string;
+  authorization_rules: Array<AuthorizationRelationToAPI> | null;
+  endpoint: EndpointToAPI;
+  is_enabled: boolean | null;
+}
+
+export interface AuthConditions {
+  attribute_path: string;
+  authorized_values: Array<string> | null;
+  blacklist_client_addresses: Array<string>;
+  endpoint: EndpointToAPI;
+  is_enabled: boolean | null;
```

Comment:
```suggestion
  is_enabled: boolean;
```
should always be a boolean